### PR TITLE
Updated description for DIPY

### PR DESCRIPTION
**Pull request by Garyfallidis**
Here you are @yarikoptic I hope this is good :)

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -31,31 +31,33 @@ Recommends: python-matplotlib,
...
-  - Warping tractographies into another space, e.g. MNI space
-  - Reading many different file formats, e.g. Trackvis or NIfTI
-  - Dealing with huge tractographies without memory restrictions
-  - Playing with datasets interactively without storing
 
+Description: A Pythonic library for the analysis of diffusion MRI datasets.
+
+DIPY is a free and open source software project for computational neuroanatomy. It focuses on diffusion magnetic resonance imaging (dMRI) analysis and tractography but also contains implementations of other computational imaging methods such as denoising and registration that are applicable to the greater medical imaging and image processing communities. Additionally, DIPY is an international project which brings together scientists across labs and countries to share their state-of-the-art code and expertise in the same codebase, accelerating scientific research in medical imaging.
+
+Here are some of the highlights:
```
Comment (yarikoptic) [inline debian/control:39]: FWIW -- that awkward formatting was for a reason ;)  but ok -- thanks -- I will merge and adjust it back a bit. Cheers

**Comment by Garyfallidis** [comment]
Oops sorry for the formatting. What is the standard? So that I know for the next time?

**Comment by yarikoptic** [comment]
On Mon, 22 Feb 2016, Eleftherios Garyfallidis wrote:

>    Oops sorry for the formatting. What is the standard? So that I know for
>    the next time?

this is the bible:
https://www.debian.org/doc/debian-policy/ch-controlfields.html
but I don't it is worthwhile of your  time ATM.  The main point is that
description should have stayed as a single paragraph, indented away and
separate sub-paragraphs split with a "." line.  itemized list should
also be indented a bit further

## Cheers

Yaroslav O. Halchenko
Center for Open Neuroscience     http://centerforopenneuroscience.org
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik

**Comment by Garyfallidis** [comment]
Okay thx. For the link. Will try to remember next time. Let me know how your bots respond and if there are any issues in Debian with Dipy 0.11.0

**Comment by Garyfallidis** [comment]
Did the installation worked fine in Neurodebian?

**Comment by yarikoptic** [comment]
Ah, you aren't following NeuroDebian on twitter!! shame on you! ;)  Yes -- worked out smooth as butter (besides very old debian/ubuntu releases as expected)

```
dipy_0.11.0-1_amd64.build       OK      11:00.79 real, 780.57 user, 18.94 sys, 3149304 out
dipy_0.11.0-1~nd70+1_i386.build FAILED  0:27.88 real, 5.33 user, 4.36 sys, 80288 out
dipy_0.11.0-1~nd70+1_amd64.build        FAILED  0:27.57 real, 4.66 user, 4.42 sys, 85160 out
dipy_0.11.0-1~nd80+1_i386.build OK      17:10.38 real, 1208.93 user, 38.27 sys, 2198976 out
dipy_0.11.0-1~nd80+1_amd64.build        OK      12:38.24 real, 960.50 user, 23.92 sys, 2873824 out
dipy_0.11.0-1~nd90+1_i386.build OK      16:37.11 real, 1165.34 user, 38.37 sys, 2520096 out
dipy_0.11.0-1~nd90+1_amd64.build        OK      11:15.61 real, 758.29 user, 24.64 sys, 3134376 out
dipy_0.11.0-1~nd+1_i386.build   OK      16:39.96 real, 1127.46 user, 38.62 sys, 2536360 out
dipy_0.11.0-1~nd+1_amd64.build  OK      11:13.24 real, 747.00 user, 25.16 sys, 3148288 out
dipy_0.11.0-1~nd12.04+1_i386.build      FAILED  0:21.42 real, 4.71 user, 3.17 sys, 81768 out
dipy_0.11.0-1~nd12.04+1_amd64.build     FAILED  0:22.62 real, 6.38 user, 3.27 sys, 137216 out
dipy_0.11.0-1~nd14.04+1_i386.build      OK      17:19.28 real, 1177.78 user, 36.40 sys, 2402688 out
dipy_0.11.0-1~nd14.04+1_amd64.build     OK      12:39.76 real, 852.22 user, 21.13 sys, 3062112 out
dipy_0.11.0-1~nd15.04+1_i386.build      OK      16:24.35 real, 1156.69 user, 32.06 sys, 2190008 out
dipy_0.11.0-1~nd15.04+1_amd64.build     OK      12:09.37 real, 865.48 user, 22.96 sys, 2868968 out
dipy_0.11.0-1~nd15.10+1_i386.build      OK      16:40.29 real, 1172.27 user, 32.60 sys, 2305312 out
dipy_0.11.0-1~nd15.10+1_amd64.build     OK      12:00.83 real, 818.86 user, 21.96 sys, 2906184 out

```

**Comment by Garyfallidis** [comment]
I am following you but don't check twitter so often. Good to know that all was good. :) This release was better than before. It's seems we are improving. Thanks again! And thx for the tweet too :dancers:

**Comment by yarikoptic** [comment]
glad to help! ;) Cheers!

**Pull request merged**